src/App.tsx

```diff
@@ -1,61 +1,106 @@
 import './App.scss';
+import users from './api/users';
+import todosFromServer from './api/todos';
+import { Todo } from './types/Todo';
+import { useState } from 'react';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+const intialTodos: Todo[] = todosFromServer;
```

Comment:
There is a typo in the variable name `intialTodos`. It should be `initialTodos` to correctly reflect its purpose and avoid confusion.